src/ui/map.js

```diff
@@ -1795,19 +1944,35 @@ class Map extends Camera {
     /**
      * Sets the filter for the specified style layer.
      *
+     * Filters control which features a style layer renders from its source.
+     * Any feature for which the filter expression evaluates to true will be
```

Comment:
For consistency from what I read through, would it make sense to wrap true in `.